src/Common/Uuid/UuidRegistry.php

```diff
@@ -241,7 +273,11 @@ public function createMissingUuids()
                         $prependAnd = true;
                     }
                     // First, see if it has already been set
-                    $setUuid = sqlQueryNoLog("SELECT `uuid` FROM `" . $this->table_name . "` WHERE `uuid` IS NOT NULL AND `uuid` != '' AND `uuid` != '\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0' AND " . $stringQuery, $arrayQuery)['uuid'];
+                    $sqlStatement = "SELECT `uuid` FROM `" . $this->table_name . "` WHERE `uuid` IS NOT NULL AND `uuid` != '' AND `uuid` != '\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0' AND " . $stringQuery;
+                    $setUuid = sqlQueryNoLog($sqlStatement, $arrayQuery);
+                    // original code here was not checking for whether a result was returned, not sure how it ever worked
+                    // as it should have thrown errors all over the place.
+                    $setUuid = $setUuid === false ? false : $setUuid['uuid'];
```

Comment:
Are these several lines to improve performance or to fix something?
Note that the line that was removed has `['uuid']` to the right of it, so then $setUuid will only by not empty if something is returned.
if it returned a uuid then -> $setUuid would equal the uuid
if it did not return a uuid then -> $setUuid would be empty

Reply:
It was to fix something.  If the SQL had an empty result set it returns false.  The line then becomes $setUuid = false['uuid'];  On my php setup this was throwing warnings which slows down php.  You can't take an array offset of an unknown key on a boolean.

Reply:
```
You can't take an array offset of an unknown key on a boolean.
```
yep, that doesn't sound very good :) (agree not good practice and surprised my php8 testing didn't catch that)